src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.R.Host.Client;
 
 namespace Microsoft.R.Host.Client.Session {
     public static class RSessionEvaluationCommands {
```

Comment:
This is becoming a dumping ground for all kind of stuff. Perhaps not for this PR (up to you), but I think in the future we should try to split it up. For things that are just straightforward wrappers around `rtvs` members, I think we should have separate extension classes, one for each .R file that we have in the package.